packages/wrangler/src/publish/publish.ts

```diff
@@ -458,6 +461,21 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 			);
 		}
 
+		// If we are using Constellation bindings, and are not bundling the worker
+		// we should error here as the Constellation shim won't be added
+		const betaConstellationShims = identifyConstellationBindingsAsBeta(
+			config.constellation
+		);
+		if (
+			Array.isArray(betaConstellationShims) &&
+			betaConstellationShims.length > 0 &&
+			props.noBundle
+		) {
+			throw new Error(
+				"While in beta, you cannot use Constellation bindings without bundling your worker. Please remove `no_bundle` from your wrangler.toml file or remove the `--no-bundle` flag to access Constellation bindings."
```

Comment:
This says "while in beta"

This says "alpha" https://github.com/cloudflare/workers-sdk/pull/3157/files#diff-0ec779a40744e36b45a9b28dee06739afa8aa2f00bb9388b68556e7fd87ab99eR2030

Reply:
Removed references to alpha.